corehq/apps/reports/views.py

```diff
@@ -1242,6 +1244,12 @@ def dispatch(self, request, *args, **kwargs):
                           "Sorry, we couldn't find that case. If you think this "
                           "is a mistake please report an issue.")
             return HttpResponseRedirect(CaseListReport.get_url(domain=self.domain))
+        if not user_can_access_location_id(
+            self.domain,
+            request.couch_user,
+            self.case_instance.owner_id
+        ):
```

Comment:
@esoergel what's `report` referencing in this case? (realized my comment was pending, as I'd started a review) :cry: